Stripe/StripeiOS/Source/STPAddCardViewController.swift

```diff
@@ -810,7 +824,9 @@ public class STPAddCardViewController: STPCoreTableViewController, STPAddressVie
         super.viewWillTransition(to: size, with: coordinator)
         let orientation = UIDevice.current.orientation
         if orientation.isPortrait || orientation.isLandscape {
-            cardScanner?.deviceOrientation = orientation
+            if #available(macCatalyst 14.0, *) {
```

Comment:
Should we wrap the whole method impl with this instead? I don't think any of it is relevant if scanning isn't available right?